Kernel_23/include/CGAL/Kernel/global_functions_2.h

```diff
@@ -355,6 +355,8 @@ compare_slope(const Segment_2<K> &s1, const Segment_2<K> &s2)
   return internal::compare_slope(s1, s2, K());
 }
 
+
+#ifndef CGAL_NO_DEPRECATED_CODE
```

Comment:
Its `CGAL_DEPRECATED` and `CGAL_DEPRECATED_MSG`, from `<CGAL/config.h>`.

Maybe that is not yet documented.